### PR TITLE
add https-verify, add debian package

### PR DESCRIPTION
**Pull request by BeryJu**
I've altered the config path environment variable to CONFIG_FILE, added Debian package meta data. The package is now also available on `apt.beryju.org` as a Debian package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ After that Zabbix-Cachet periodically checks Zabbix triggers (that linked to you
 * Zabbix 2.X (tested only 2.4), 3.0
 
 # Installation
+
+## apt
+1. Add the BeryJu.org APT Repository: `wget -O - -q https://apt.beryju.org/install.sh | bash`
```
Comment (qk4l) [inline README.md]: I'm sorry but it's non secure to run scripts from public source. Please change it to 
```
wget -O - -q https://apt.beryju.org/public.key | apt-key add - 
echo "deb http://apt.beryju.org/ stable beryjuorg" > /etc/apt/sources.list.d/beryjuorg.list
apt-get update -qq 
 ```

File: config-example.yml [hunk 2]
```diff
@@ -1,22 +0,0 @@
...
-  # Leave it empty if you want to use /
-  root_service: 'Status Page'
-  # How often check Zabbix for new incidents
-  update_inc_interval: 120  # in seconds
-  # How often check Zabbix for new IT Services
-  update_comp_interval: 3600  # in seconds
-  # Log level https://docs.python.org/3.4/library/logging.html#levels
-  log_level: INFO
-  # Additional logging level for requests module
-  log_level_requests: WARNING
```
Comment (qk4l) [inline config-example.yml]: It's example config file. Why you deleted it?

**Comment by BeryJu** [comment]
No idea why the config got lost, I just had it renamed to config.yml for packaging. Oh well.

**Comment by qk4l** [comment]
@BeryJu Great work, thanks.

**Pull request merged**